README.md

```diff
@@ -69,6 +69,11 @@ Documentation (built daily from `master`) is available here:
 
 https://jwst-pipeline.readthedocs.io/en/latest/
 
+One can clone this repository and build the documentation with
+
+    pip install stsci_rtd_theme
```

Comment:
They may also need to pip install `sphinx_rtd_theme` and  `sphinx-automodapi`

Reply:
👍 on @sosey comment.